cloudflare.go

```diff
@@ -484,6 +494,52 @@ func fetchAccounts() []cfaccounts.Account {
 	return cfAccounts
 }
 
+func fetchZone5mStats(zoneIDs []string) (*cloudflareResponse, error) {
+	request := graphql.NewRequest(`
+query ($zoneIDs: [String!], $mintime: Time!, $maxtime: Time!, $limit: Int!) {
+  viewer {
+    zones(filter: { zoneTag_in: $zoneIDs }) {
+	 zoneTag
+     httpRequestsAdaptiveGroups(filter:{datetime_gt: mintime, datetime_lt: $maxtime}, limit: $limit) {
```

Comment:
```suggestion
     httpRequestsAdaptiveGroups(filter:{datetime_gt: $mintime, datetime_lt: $maxtime}, limit: $limit) {
```

Dollar bill on mintime too?